src/tools/csg_stat_imc.cc

```diff
@@ -136,11 +163,30 @@ Imc::interaction_t *Imc::AddInteraction(Property *p)
     i->_max = p->get("max").as<double>();
     i->_norm = 1.0;
     i->_p=p;
+    i->_threebody=0;
+    i->_force=0;    
+    //check if option threebody exists
+    if (p->exists("threebody")) {
+        i->_threebody = p->get("threebody").as<bool>();            
+    }
+    //check if option force exists
+    if (p->exists("force")) {
+        i->_force = p->get("force").as<bool>();            
+    }    
+    //preliminary. should be changed
+    i->_cut = 0.37; //(0.37 nm)
```

Comment:
tools has now a function ```ifExistsReturnElseReturnDefault``` which does all this a bit shorter

Reply:
changed